### PR TITLE
Updates to Annotations 

### PR DESCRIPTION
**Pull request by hughbrien**
Updated the Annotations Documentation to add some clarification and specificity

### CODE REVIEW

File: docs/Learn/Annotations.md [hunk 1]
```diff
@@ -1,47 +1,103 @@
-# Komodor kubernetes annotations
+# Komodor Kubernetes Annotations
 
-Komodor annotations (AKA Komodor as Code), is a method to allow users to configure everything related to Komodor as part of their native k8s yaml.
-Komodor annotations should be placed in the deployment resource annotations (annotations set on the pod template are ignored)
+Komodor annotations enable users to create static and dynamic hyperlinks
+to other solutions and resources. For example, you can use Annotations to provide hyperlinks to the following
 
+- Source Control like Git,
```
Comment (ronahk) [inline docs/Learn/Annotations.md:6]: Is the comma here correct?

File: docs/Learn/Annotations.md [hunk 2]
```diff
@@ -1,47 +1,103 @@
...
+- DevOps tools,
```
Comment (ronahk) [inline docs/Learn/Annotations.md:7]: Same here

File: docs/Learn/Annotations.md [hunk 3]
```diff
@@ -1,47 +1,103 @@
...
+to other solutions and resources. For example, you can use Annotations to provide hyperlinks to the following
 
+- Source Control like Git,
+- DevOps tools, 
+- Monitoring and observability
+- Integrated solutions
 
-## CI-Deploy Links
+**Where does Komodor utilize annotations?**
+Komodor uses these annotations enrich service information and creates powerful connections between services.
```
Comment (ronahk) [inline docs/Learn/Annotations.md:12]: to enrich?

File: docs/Learn/Annotations.md [hunk 4]
```diff
@@ -1,47 +1,103 @@
...
+- The **Info** tab of the Service 
+- The **Deployment Details** of a Deployment Event 
+
+
+Komodor annotations should be placed in the deployment resource annotations (annotations set on the pod template are ignored)
+
+
+## Annotation Links
+
+Annotation links have  different types:
```
Comment (ronahk) [inline docs/Learn/Annotations.md:50]: extra space here

**Review by ronahk — COMMENTED** [review]
Looks good, some minor comments/questions.

File: docs/Learn/Annotations.md (hunk 1 above)
Comment (hughbrien) [reply docs/Learn/Annotations.md]: Removed

File: docs/Learn/Annotations.md (hunk 2 above)
Comment (hughbrien) [reply docs/Learn/Annotations.md]: Removed from original source

File: docs/Learn/Annotations.md (hunk 3 above)
Comment (hughbrien) [reply docs/Learn/Annotations.md:12]: If you want to removed it you can but here is the meaning : 

enrich : transitive verb : To make fuller, more meaningful, or more rewarding.

Comment (hughbrien) [reply docs/Learn/Annotations.md:12]: Yes. if it's an obvious typo please fix it : )

**Review by ronahk — APPROVED** [review]

File: docs/Learn/Annotations.md [hunk 5]
```diff
@@ -1,47 +1,106 @@
-# Komodor kubernetes annotations
+# Komodor Kubernetes Annotations
 
-Komodor annotations (AKA Komodor as Code), is a method to allow users to configure everything related to Komodor as part of their native k8s yaml.
-Komodor annotations should be placed in the deployment resource annotations (annotations set on the pod template are ignored)
+Komodor annotations enable users to create static and dynamic hyperlinks to other solutions and resources.
```
Comment (gxxglyeyez) [inline docs/Learn/Annotations.md:3]: @hughbrien I don't think that we should limit it only to hyperlinks.
We might want to add alert configuration via annotations or custom filters

**Review by gxxglyeyez — CHANGES_REQUESTED** [review]

File: docs/Learn/Annotations.md (hunk 5 above)
Comment (hughbrien) [reply docs/Learn/Annotations.md:3]: I am not sure what this means. Are saying we can "link to screens in Komodor"?  or other Solutions? 
"We might want to add alert configuration via annotations or custom filters"

**Comment by hughbrien** [comment]
Can We move these changes forward.

**Comment by hughbrien** [comment]
I would like to get changes published.  If you have an issue with any of them. Just reject or update them.

**Review by gxxglyeyez — APPROVED** [review]

**Pull request merged**